src/apps/material/helper.ts

```diff
@@ -107,6 +107,23 @@ export const getManifestationLanguages = (manifestation: Manifestation) => {
   );
 };
 
+export const getManifestationLanguageIsoCode = (
+  manifestation: Manifestation[]
+) => {
+  const uniqueLanguages = uniqBy(
+    manifestation.map((m) => m.languages),
+    "main[0].isoCode"
+  );
+  if (
+    uniqueLanguages.length === 1 &&
+    uniqueLanguages[0]?.main?.[0]?.isoCode &&
+    uniqueLanguages[0].main[0].isoCode !== "dan"
+  ) {
+    return uniqueLanguages[0].main[0].isoCode;
+  }
+  return undefined;
```

Comment:
Please reconsider if it is appropriate to return `undefined` if the language is Danish and if we only want to return a language if the manifestations only contain one language.

I understand what you are trying to do but I do not see it doing any harm specifying Danish as a language for titles in Danish. If the system is already in Danish it should be a no-op. If we get to translating the system to English or German then it will provide value to have Danish titles read out loud in Danish.

As I see it this will make the code easier to understand. This is especially important as the function stands without further documentation or reasoning.